orchestrator/gorc/src/commands/cosmos_to_eth.rs

```diff
@@ -60,121 +57,112 @@ impl Runnable for CosmosToEthCmd {
+                ),
+                Err(_e) => {
+                    println!(
+                        "Asset {} has no ERC20 representation, you may need to deploy an ERC20 for it!",
+                        gravity_denom
+                    );
+                    exit(1);
+                }
             }
-        }
 
-        println!("Cosmos balances {:?}", balances);
-
-        let times = self.args.get(4).expect("times is required");
-        let times = times.parse::<usize>().expect("cannot parse times");
+            let amount = Coin {
+                amount: amount.clone(),
+                denom: gravity_denom.clone(),
+            };
+            let bridge_fee = Coin {
+                amount: 1u64.into(),
+                denom: gravity_denom.clone(),
+            };
+
+            let eth_dest = self.args.get(3).expect("ethereum destination is required");
+            let eth_dest: EthAddress = eth_dest.parse().expect("cannot parse ethereum address");
+            check_for_fee_denom(&gravity_denom, cosmos_address, &contact).await;
+
+            let balances = contact
+                .get_balances(cosmos_address)
+                .await
+                .expect("Failed to get balances!");
+            let mut found = None;
+            for coin in balances.iter() {
+                if coin.denom == gravity_denom {
+                    found = Some(coin);
+                }
+            }
 
-        match found {
-            None => panic!("You don't have any {} tokens!", gravity_denom),
-            Some(found) => {
-                if amount.amount.clone() * times.into() >= found.amount && times == 1 {
-                    if is_cosmos_originated {
-                        panic!("Your transfer of {} {} tokens is greater than your balance of {} tokens. Remember you need some to pay for fees!", print_atom(amount.amount), gravity_denom, print_atom(found.amount.clone()));
-                    } else {
-                        panic!("Your transfer of {} {} tokens is greater than your balance of {} tokens. Remember you need some to pay for fees!", print_eth(amount.amount), gravity_denom, print_eth(found.amount.clone()));
-                    }
-                } else if amount.amount.clone() * times.into() >= found.amount {
-                    if is_cosmos_originated {
-                        panic!("Your transfer of {} * {} {} tokens is greater than your balance of {} tokens. Try to reduce the amount or the --times parameter", print_atom(amount.amount), times, gravity_denom, print_atom(found.amount.clone()));
-                    } else {
-                        panic!("Your transfer of {} * {} {} tokens is greater than your balance of {} tokens. Try to reduce the amount or the --times parameter", print_eth(amount.amount), times, gravity_denom, print_eth(found.amount.clone()));
+            println!("Cosmos balances {:?}", balances);
+
+            let times = self.args.get(4).expect("times is required");
```

Comment:
I honestly think we should delete this `times` thing, it seems absolutely useless and just makes this more complicated.